### ISSUE TITLE
SyntaxError: Unexpected token ')'  clipboard.js:17 in 2.0.7

### ISSUE THREAD

**Issue by mrch81** [zenorocha/clipboard.js#738]
## 🐛 Bug Report

In dist/clipboard.js in v2.0.7, there's this bizarre number  `134:`  on line 20

The reference file here: 
https://github.com/zenorocha/clipboard.js/blob/c38b4ee76f1e9bacbf2f019e9e84f8f2e2407e91/dist/clipboard.js

It seems to cause the `SyntaxError: Unexpected token ')' ` during testing with PhantomJS 2.1.1
### Expected Behaviour

In v2.0.6, the grunt test with PhantomJS 2.1.1 passed without any error.
Now it fails with `SyntaxError: Unexpected token ')' `

**Comment by vitormalencar** [comment]
this is probably related to the  #737 
also, this line number is created by webpack

**Comment by vitormalencar** [comment]
So @mrch81  I just took some time today to create a new release, I included the webpack changes I hope it helps to solve the issue, if not I will continue to push more fixes =)

Please let me know if the issue is solved or not

**Comment by mrch81** [comment]
Thanks, that was rather fast. Unfortunately that fix doesn't resolve the issue :(

Perhaps the details below may help:

With **clipboard#2.0.8**
```
11 03 2021 05:05:16.943:INFO [karma]: Karma v1.7.1 server started at http://0.0.0.0:8080/
11 03 2021 05:05:16.943:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
11 03 2021 05:05:16.960:INFO [launcher]: Starting browser PhantomJS
11 03 2021 05:05:18.308:INFO [PhantomJS 2.1.1 (Linux 0.0.0)]: Connected on socket 2VxU7lpGcyFXyR6pAAAA with id 87091729
PhantomJS 2.1.1 (Linux 0.0.0) ERROR
  SyntaxError: Unexpected token ')'
  at bower_components/clipboard/dist/clipboard.js:17

Warning: Task "karma:mhsi" failed. Use --force to continue.

Aborted due to warnings.
```

With **clipboard#2.0.6**

```
11 03 2021 05:09:18.948:INFO [karma]: Karma v1.7.1 server started at http://0.0.0.0:8080/
11 03 2021 05:09:18.949:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
11 03 2021 05:09:18.963:INFO [launcher]: Starting browser PhantomJS
11 03 2021 05:09:19.289:INFO [PhantomJS 2.1.1 (Linux 0.0.0)]: Connected on socket K4bxvJOelyYpKw2TAAAA with id 61694027
PhantomJS 2.1.1 (Linux 0.0.0): Executed 108 of 108 SUCCESS (5.484 secs / 6.698 secs)

Done.

```

**Testing framework:** Jasmine

Please do let me know if I could provide further information.

**Comment by vitormalencar** [comment]
@mrch81 thanks a lot for the log, is quite strange since that line 17 is basically webpack module initialization,

can you maybe put a small example of this test on Github, or some other code platform, so I can test a little bit more ?

**Comment by mrch81** [comment]
@vitormalencar Please find it attached here: [clipboard.zip](https://github.com/zenorocha/clipboard.js/files/6123214/clipboard.zip)

The zip contains the `bower_components/clipboard/dist/clipboard.js` 

Also, let me know if you need the one with 2.0.6.

**Comment by vitormalencar** [comment]
@mrch81 thanks a lot this is what I was looking for, the problem is that we now avoid commit the dist to the project so in fact you are using v2.0.7 still , but is still required  if you are using bower, the fix here is easy I just need to publish the dist on the main repo and I will ask you to try again after that

**Comment by vitormalencar** [comment]
@mrch81 yes I just confirmed that this was the problem, bower users I will probably do another release end of next week that should fix this issue, in the meantime, you can still use `v2.0.6` with no problems since the release was just to fix some minor bugs, or you can switch to use npm and use the latest version `2.0.8`

**Comment by mrch81** [comment]
Thanks @vitormalencar I've patched my env with 2.0.6.

I will keep an eye out for the fix and test it as soon as it is available.
